ams/server/pom.xml

```diff
@@ -219,6 +219,42 @@
             <artifactId>kyuubi-hive-jdbc-shaded</artifactId>
         </dependency>
 
+        <!-- aws dependencies -->
+        <dependency>
```

Comment:
I am not sure why specific AWS dependencies are only added to the `ams-server` module. Are these dependencies not needed for other modules, such as the `StandaloneOptimizer`?

Reply:
StandaloneOptimizer  is not a fatjar and event not a dependent plugin. It was put to ./lib and using the AMS classpath.

aws dependencies should be controlled by engines.